docs/index.md

```diff
@@ -0,0 +1,41 @@
+# dbt_project_meshify
```

Comment:
What was the reason for this header title?

Reply:
this was a copy paste error from evaluator 😅 updated to say dbt_meshify (also open to removing)